### ISSUE TITLE
static_assert at the end

### ISSUE THREAD

**Issue by swwwolf** [MarkHC/windbg_to_c#3]
I'm suggesting to add a static_assert after generated structure to check it's size.

static_assert(sizeof(BLAH_BLAH_BLAH) == 0x20, "Invalid BLAH_BLAH_BLAH size");

Also, you can use intellectually (somehow) choosen asserts like:

static_assert(FIELD_OFFSET(BLAH_BLAH_BLAH, SomeMember) == 0x18, "Invalid SomeMember offset");

**Comment by markhc** [comment]
Yes I had planned to add something like that but never got around to it.

I'll see what I can do.
